Packs/rasterize/Integrations/rasterize/rasterize.py

```diff
@@ -820,21 +826,24 @@ def perform_rasterize(path: str | list[str],
             demisto.info(
                 f"Finished {len(rasterization_threads)} rasterize operations, active tabs len: {len(browser.list_tab())}")
 
-            previous_rasterizations_counter_from_file = read_file(RASTERIZATIONS_COUNTER_FILE_PATH)
+            previous_rasterizations_counter_from_file: dict = read_json_file()
             if previous_rasterizations_counter_from_file:
-                total_rasterizations_count = int(previous_rasterizations_counter_from_file) + len(rasterization_threads)
+                per_port_rasterizations_count = previous_rasterizations_counter_from_file[chrome_port]['rasterize_count'] + len(
+                    rasterization_threads)
             else:
-                total_rasterizations_count = len(rasterization_threads)
-            demisto.debug(f"Should Chrome be terminated?, {total_rasterizations_count=},"
+                # check this else with Tal
```

Comment:
?

Reply:
I'm waiting for @tkatzir for respond if we should lease the thread count or not- currently this comment is for marking.